src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesArray = stylesString.split(';');
+  const stylesObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `stylesObject`. Descriptive names improve code readability. Consider a name that reflects the purpose of the object, such as `styleObject` or `cssStyles`.